### DISCUSSION TITLE
Times downed edit

### DISCUSSION THREAD

**Discussion by AdamTelos** [AnthonyMichaelTDM/DRG-Save-Editor#38] [Ideas]
Hello,

Just wondering, how difficult would it be to add a feature where you can edit "Times downed" - there's a bug for a lot of people who play deep rock where the number displays wrong, so it would be nice to be able to reset it in a sense, and keep track of it from there.

**Comment by AnthonyMichaelTDM** [comment]
It’d involve finding where that value is stored in the save file, adding a new section to the gui, and some other book keepering

**Reply by AdamTelos** [reply]
Thank you, I might try doing this. I suppose I'll need to see what changes every time I get downed. Any advice would be appreciated. My experience with this save file is limited to using a hex editor to edit skins in, but there was already a good guide for that.

**Comment by AnthonyMichaelTDM** [comment]
Awesome, I let me know if you have any more questions. If you get it working and want to make a PR, I’d  really appreciate it

**Reply by AdamTelos** [reply]
Hello!

I got a little time to look into this, results unusual. Maybe this will help you a bit but I'm unsure.

So I basically started a new save file (perhaps unwise), skipped the tutorial and decided to repeatedly enter a mission, and use my throwables on myself until I down myself, then abort mission. I did this 5 times and used HxD and checked which offsets changed each time.

After this I did 1 more mission where I entered and simply aborted the mission (so times downed wouldn't increment). I then checked my list of offsets and removed the ones that changed, as they shouldn't have. I was left with just 2.

625
7E7

I kept all my save files from 0 downed to 5 downed as well as the 5 downed + 1 abort without downing.
I copied my value from 0 downed to the 625 offset on my save and the game failed to load
I then copied my value from 0 downed to the 7E7 offset and my times downed changed from 5 to 4.

So I haven't managed to solve this as I'm a noob but let me know if anything here is unclear or if I can provide any save files or screenshots that might help. I'm pretty confused as to why it went to 5 to 4 instead of 5 to 0 so I'm just puzzled and unsure how to proceed tbh.

**Reply by AnthonyMichaelTDM** [reply]
you're probably on the right track, maybe both of those offsets are used somehow? I'm not really sure and i'm not in the position to figure it out myself right now (college be crazy)

**Reply by AdamTelos** [reply]
Ah, good luck with college. I'm glad college is behind me :)

I'm basically stuck too. There's a bunch of stuff I don't know (How to tell when parts of the savefile stop referring to 1 thing and start referring to another for example). If you ever end up having more free time do let me know and I'll provide the info I have (revivetest on desktop folder for my own reference)
